MusicCat-Console/Program.cs

```diff
@@ -1,29 +1,72 @@
 ﻿using System;
+using System.IO;
 using System.Threading;
+using ApiListener;
+using MusicCat;
+using MusicCat.Metadata;
+using Newtonsoft.Json;
 
 namespace ConsoleWrapper
 {
     class Program
     {
-        const ApiListener.ApiLogLevel displayLogLevel = ApiListener.ApiLogLevel.Debug;
+        const ApiLogLevel displayLogLevel = ApiLogLevel.Debug;
 
         static void Main(string[] args)
         {
+	        FileStream logStream = null;
+	        StreamWriter logWriter = null;
             var monitor = new object();
-            //TODO: Load config
-            MusicCat.Listener.Config = MusicCat.Config.DefaultConfig;
-            MusicCat.Listener.AttachLogger(m =>
+	        string configJson = File.ReadAllText("MusicCatConfig.json");
```

Comment:
This being outside the try catch means if the JSON file isn't present, the program crashes.